python/module/CMakeLists.txt

```diff
@@ -6,8 +6,13 @@ foreach(p ${ALL_PYTHON_BINDING_TARGETS})
   target_link_libraries(${p}
     PRIVATE
     openstudiolib
-    Python::Python
   )
+  if(UNIX AND NOT APPLE)
+    target_link_libraries(${p}
+      PRIVATE
+      Python::Python
+    )
+  endif()
```

Comment:
Apparently on mac I can get by by NOT linking to libpython for the individual swig bindings (eg _openstudioairflow.so). On Ubuntu I cannot

Reply:
(we should probably figure that out at a later date, fyi @kbenne )

Reply:
Is there a linker flag to ignore unresolved symbols somewhere? That would be my first suspicion.

Reply:
https://github.com/NREL/OpenStudio/blob/65bcf102dbde3fe69d83d5ef61ed09694588e925/ProjectMacros.cmake#L380-L381